editions/tw5.com/tiddlers/commands/RenderTiddlerCommand.tid

```diff
@@ -1,6 +1,9 @@
-title: RenderTiddlerCommand
-tags: Commands
 caption: rendertiddler
+created: 20241007001432813
+modified: 20241007001436537
```

Comment:
It would be better to omit the created and modified fields for these deprecated commands to prevent them from rising to the top of the "Recent" tab